geoutils/raster/raster.py

```diff
@@ -3272,7 +3512,7 @@ def interp_points(
             transformer = pyproj.Transformer.from_crs(init_crs, dest_crs)
             x, y = transformer.transform(x, y)
 
-        i, j = self.xy2ij(x, y, op=np.float32, shift_area_or_point=shift_area_or_point)
+        i, j = self.xy2ij(x, y, shift_area_or_point=shift_area_or_point)
```

Comment:
why has the argument `op` been removed?

Reply:
it's the function default, so we don't need to specify!